CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [12.0.1]
+### Changed
+- Disabled the "Restore Backup" feature ([#26338](https://github.com/MetaMask/metamask-extension/pull/26338))
```

Comment:
Nit "removed" seems like a better description, removed is for removed features. I added a bit more detail on the reasoning as well.
```suggestion
### Removed
- Disabled the "Restore Backup" feature ([#26338](https://github.com/MetaMask/metamask-extension/pull/26338))
  - This was linked to reports of state corruption. It has been disabled at least until we can ensure it is safe to use.
```

Reply:
We should also update this to use the original PR number:

26325

```suggestion
### Removed
- Disabled the "Restore Backup" feature ([#26325](https://github.com/MetaMask/metamask-extension/pull/26325))
  - This was linked to reports of state corruption. It has been disabled at least until we can ensure it is safe to use.
```